05_case_study.Rmd

```diff
@@ -0,0 +1,379 @@
+---
+title: "A predictive modeling case study"
+output: 
+  html_document:
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, message = FALSE, warning = FALSE)
+options(tibble.print_min = 5)
+```
+
+`r shiny::icon('map-signs')` Get started with building a model in this R Markdown document that accompanies [A predictive modeling case study](https://www.tidymodels.org/start/case-study/) tidymodels start article.
+
+`r shiny::icon('map')` If you ever get lost, you can click on the header of the knitted document to see the accompanying section in the online article.
+
+`r shiny::icon('gear')` Take advantage of the RStudio IDE and use "Run All Chunks Above" or "Run Current Chunk" buttons to easily execute code chunks.
+
+
+## [Introduction](https://www.tidymodels.org/start/models/#intro)
+
+Let's put everything we learned from each of the previous [Get Started](https://www.tidymodels.org/start/) articles together and build a predictive model from beginning to end with data on hotel stays. 
+
+Load necessary packages:
+
+```{r}
+library(tidymodels)  
+
+# Helper packages
+library(readr)       # for importing data
+library(vip)         # for variable importance plots
+```
+
+## [The Hotel Bookings Data](https://www.tidymodels.org/start/case-study/#data)
+
+Let's read our hotel data into R:
+
+```{r}
+hotels <- 
+  read_csv('https://tidymodels.org/start/case-study/hotels.csv') %>%
+  mutate_if(is.character, as.factor) 
+
+dim(hotels)
+glimpse(hotels)
+```
+
+Let's look at proportions of hotel stays that include children and/or babies:
+
+```{r}
+hotels %>% 
+  count(children) %>% 
+  mutate(prop = n/sum(n))
+```
+
+
+## [A first model: penalized logistic regression](https://www.tidymodels.org/start/case-study/#first-model)
+
+ Recall [Evaluate your model with resampling](/start/resampling/#data-split) article for data splitting?
+ 
+ Let's reserve 25% of the stays to the test set:
+ 
+```{r}
+set.seed(123)
+splits      <- initial_split(hotels, strata = children)
+
+hotel_other <- training(splits)
+hotel_test  <- testing(splits)
+
+# training set proportions by children
+hotel_other %>% 
+  count(children) %>% 
+  mutate(prop = n/sum(n))
+
+# test set proportions by children
+hotel_test  %>% 
+  count(children) %>% 
+  mutate(prop = n/sum(n))
+```
+
+Now let's reserve another 20% of the `hotel_other` for our validation set.
+
+```{r}
+set.seed(234)
+val_set <- validation_split(hotel_other, 
+                            strata = children, 
+                            prop = 0.80)
+val_set
+```
+ 
+
+### Build the model
+
+Let's specify a penalized logistic regression model using the lasso method.
+Note that we define `penalty=tune()` so we can tune it in the next steps, and set `mixture=1` so we are using the lasso method. 
```

Comment:
```suggestion
Note that we define `penalty = tune()` so we can tune it in the next steps, and set `mixture=1` so we are using the lasso method. 
```

Reply:
changed to
```
Note that we define `penalty = tune()` so we can tune it in the next steps, and since we are using lasso method, we set `mixture = 1`.
```